src/plugins/index_pattern_field_editor/public/components/field_editor/form_fields/script_field.tsx

```diff
@@ -91,7 +91,10 @@ export const ScriptField = React.memo(({ existingConcreteFields, links, syntaxEr
                 const painlessSyntaxErrors = PainlessLang.getSyntaxErrors();
                 // It is possible for there to be more than one editor in a view,
                 // so we need to get the syntax errors based on the editor (aka model) ID
-                const editorHasSyntaxErrors = editorId && painlessSyntaxErrors[editorId].length > 0;
+                const editorHasSyntaxErrors =
```

Comment:
Sometimes `painlessSyntaxErrors` doesn't have an `editorId` (I suspect timing issues) - this crashes the editor in a way the save button can't be pressed